test/test_esapi.py

```diff
@@ -3,9 +3,83 @@
 SPDX-License-Identifier: BSD-2
 """
 
-import unittest
+import binascii
+
+from tpm2_pytss._libtpm2_pytss import ffi, lib
+from tpm2_pytss.ESAPI import (
+    ESYS_TR,
+    TPM2_ALG,
+    TPM2_CAP,
+    TPM2_CC,
+    TPM2_ECC_CURVE,
+    TPM2_RC,
+    TPM2_SE,
+    TPM2_SU,
+    TPM2B_ATTEST,
+    TPM2B_AUTH,
+    TPM2B_CREATION_DATA,
+    TPM2B_DATA,
+    TPM2B_DIGEST,
+    TPM2B_ECC_PARAMETER,
+    TPM2B_ECC_POINT,
+    TPM2B_ENCRYPTED_SECRET,
+    TPM2B_EVENT,
+    TPM2B_ID_OBJECT,
+    TPM2B_IV,
+    TPM2B_MAX_BUFFER,
+    TPM2B_NAME,
+    TPM2B_NONCE,
+    TPM2B_NV_PUBLIC,
+    TPM2B_PRIVATE,
+    TPM2B_PUBLIC,
+    TPM2B_PUBLIC_KEY_RSA,
+    TPM2B_SENSITIVE,
+    TPM2B_SENSITIVE_CREATE,
+    TPM2B_SENSITIVE_DATA,
+    TPM2B_TEMPLATE,
+    TPMA_OBJECT,
+    TPML_ALG,
+    TPML_CC,
+    TPML_DIGEST_VALUES,
+    TPML_PCR_SELECTION,
+    TPMS_ALGORITHM_DETAIL_ECC,
+    TPMT_HA,
+    TPMT_PUBLIC,
+    TPMT_PUBLIC_PARMS,
+    TPMT_RSA_DECRYPT,
+    TPMT_SIG_SCHEME,
+    TPMT_SIGNATURE,
+    TPMT_SYM_DEF,
+    TPMT_SYM_DEF_OBJECT,
+    TPMT_TK_CREATION,
+    TPMT_TK_HASHCHECK,
+    TPMT_TK_VERIFIED,
+)
```

Comment:
These should be imported from tpm2_pytss.types

Reply:
You're right, my bad